src/Common/src/System/Net/Logging/WebEventSource.cs

```diff
@@ -0,0 +1,82 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics.Tracing;
+using System.Globalization;
+using System.Net.Sockets;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace System.Net
+{
+    [EventSource(Name = "Microsoft-System-Net-Mail", LocalizationResources = "FxResources.System.Net.Mail.SR")]
+    internal sealed class WebEventSource : EventSource
```

Comment:
SmtpClient uses a TraceSource named as Web on Desktop, kept the same naming scheme from Desktop code [here](https://referencesource.microsoft.com/#System/net/System/Net/mail/SmtpClient.cs,71)